### PR TITLE
Connectivity Tests: Fix TRC  verification delay

### PR DESCRIPTION
**Pull request by and-rewsmith**
TRC verification delay defaults to the correct value, but pipeline injects a value too low. I am removing the plumbing for this configuration because it adds unnecessary overhead. This will also fix the verification delay.

## Azure IoT Edge PR checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines and Best Practices
- [x] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
- [x] Title of the pull request is clear and informative.
- [x] Description of the pull request includes a concise summary of the enhancement or bug fix.

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.
- Description of the pull request includes 
	- [x] concise summary of tests added/modified
	- [x] local testing done.  

### Draft PRs
- Open the PR in `Draft` mode if it is:
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.

_Note: We use the kodiakhq bot to merge PRs once the necessary checks and approvals are in place. When it merges a PR, kodiakhq converts the PR title to the commit title, PR description to the commit description, and squashes all the commits in the PR to a single commit. The net effect is that entire PR becomes a single commit. Please follow the best practices mentioned [here](https://chris.beams.io/posts/git-commit/#:~:text=The%20seven%20rules%20of%20a%20great%20Git%20commit,what%20and%20why%20vs.%20how%20For%20example%3A%20) for the PR title and description_

### CODE REVIEW

**Comment by and-rewsmith** [comment]
Will wait on connectivity before merge:
https://dev.azure.com/msazure/One/_build/results?buildId=58020671&view=results

**Comment by and-rewsmith** [comment]
New run:
https://dev.azure.com/msazure/One/_build/results?buildId=58030687&view=results

File: scripts/linux/trcE2ETest.sh
Comment (yophilav) [inline scripts/linux/trcE2ETest.sh:878]: Please add a comment that explains 'why' these two variables {`VERIFICATION_DELAY`, `CHECK_TRC_DELAY`} are needed and how are they different. 

Nit: `CHECK_TRC_DELAY` sounds misleading. Maybe we can name it something like... `1ES_AGENT_NETWORKING_DELAY`?

File: scripts/linux/trcE2ETest.sh
Comment (yophilav) [inline scripts/linux/trcE2ETest.sh:875]: I think we are asking ourselves for trouble here. Previously, the eventHub has 45 minutes to ingress and the messages before TRC create a report. Now it only has 15 minutes after the test completion to ingress.... This screams instability.

Comment (and-rewsmith) [reply scripts/linux/trcE2ETest.sh:875]: This is reverting the verification delay to what is was before my previous PRs changes, which had no effect on real verification delay.

File: scripts/linux/trcE2ETest.sh
Comment (and-rewsmith) [reply scripts/linux/trcE2ETest.sh:878]: 1. Will add better comment, good idea.
2. 1ES_AGENT_NETWORKING_DELAY indicates why it was there, but I'd rather have a name that indicates what the delay is doing. The other names follow this pattern, so I'd rather go with CHECK_TRC_DELAY.
3. Yes my intention was to bump test start delay to 10 mins.

Comment (and-rewsmith) [reply scripts/linux/trcE2ETest.sh:878]: Regarding 2, I think the comment you suggested will be the best of both worlds

**Comment by and-rewsmith** [comment]
New run:
https://dev.azure.com/msazure/One/_build/results?buildId=58030691&view=results

**Comment by and-rewsmith** [comment]
New run:
https://dev.azure.com/msazure/One/_build/results?buildId=58160908&view=results

**Review by yophilav — APPROVED** [review]

**Pull request merged**
